Source/Particles/Collision/MCCScattering.H

```diff
@@ -0,0 +1,304 @@
+/* Copyright 2021 Modern Electron
+ *
+ * This file is part of WarpX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+#ifndef WARPX_PARTICLES_COLLISION_MCC_SCATTERING_H_
+#define WARPX_PARTICLES_COLLISION_MCC_SCATTERING_H_
+
+#include "MCCProcess.H"
+#include "Utils/WarpXConst.H"
+#include <AMReX_Random.H>
```

Comment:
For type literals below:
```suggestion
#include <AMReX_Random.H>
#include <AMReX_REAL.H>
```